spec/models/shared_examples/shared_examples_for_archivable_by_parent_id.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.shared_examples 'an archivable model' do |options|
+RSpec.shared_examples 'an archivable model by parent id' do |options|
```

Comment:
Can we revert the changes in this file? (i.e. can this file and `shared_examples_for_archivable_by_version_id.rb ` be merged together?)
What is the difference between the files?

Reply:
I actually had considered this and implemented a solution that combined the files when refactoring the archiver sql and active record queries based off of your suggestion. The issue I ran into was that while the two types of models are similar the setup and tests vary slightly and require some additional conditional logic, since one type relies on `version_id` while the other uses `institution_id`.

I wasn't sure what would be the lesser of two evils in this scenario, adding conditional logic for a single test for all 4 models, or having 2 tests for the two types of models as it is currently.

Thoughts?